docs/UsersGuide/source/BuildingRunningTesting/VXCases.rst

```diff
@@ -147,7 +154,7 @@ Then, edit the configuration file (``config.yaml``) to include the variables and
          
    To modify the file, hit the ``i`` key and then make any changes required. To close and save, hit the ``esc`` key and type ``:wq``. Users may opt to use their preferred code editor instead. 
```

Comment:
I suggest putting here link to vi tutorial, rather the explaining basic options ('i', 'Esc' and ':wq').

Reply:
I think that in general, the concern with external links is that pages can be removed, and users end up at a 404 Not Found. It also takes them away from the page that they're trying to use, which isn't ideal from a UX perspective. One option might be to create and link to a mini-tutorial (as I've done for some of the SSH stuff). Then at least users are still within the docs. Will have to think on it...

Reply:
Oh, yes, that second option sounds much better.